src/main/scala/com/thirdship/libunit/units/MetricTSUnit.scala

```diff
@@ -0,0 +1,52 @@
+package com.thirdship.libunit.units
+
+import com.thirdship.libunit.utils.{ExactString, FuzzyString, WordString}
+import com.thirdship.libunit.utils.Helpers._
+import com.thirdship.libunit._
+
+class MetricTSUnit(unit: String, parseList: List[FuzzyString]) {
```

Comment:
This is the intended modification for TimeTSUnit, to give you an example:
`object TimeHelpers{
	object Seconds { def apply(value: Double = 1) = TSUnitValuePair(value, new TimeTSUnit("s" )) }
	object Minutes { def apply(value: Double = 1) = TSUnitValuePair(value, new TimeTSUnit("min")) }

	private val baseUnit = "s"

	private val baseParseList = List("sec".i, "second".w)

	val lore: MetricTSUnit = new MetricTSUnit(baseUnit,baseParseList)

	private val compressedParseMap = Map(
		baseUnit.i 	-> baseParseList,
		"min".i 	-> List("minute".w),
		"h".i 		-> List("hr".w, "hour".w),
		"d".i 		-> List("day".w),
		"y".i 		-> List("yr".w, "year".w),
		"jiffy".i 	-> List()
	) + lore.compressedParseMap

	private val edges = List(
    	new ScalarConversionEdge(baseUnit,	baseUnit,	1),
		new ScalarConversionEdge("min", 	baseUnit,	60),
		new ScalarConversionEdge("h",	 	"min",		60),
		new ScalarConversionEdge("d", 		"h",		24),
		new ScalarConversionEdge("y", 		"d",		365.2422,	1),
		new ScalarConversionEdge(baseUnit, 	"jiffy",	3e-24,		1)
	) + lore.edges

	val data: AStarConvertibleTSUnitData = new AStarConvertibleTSUnitData(baseUnit, "Time", compressedParseMap, edges)
}`
Some of this can be seen in MetricTSUnitTest. As for refactoring to a different filename, I am open to suggestions.